ping/_compositions/go-cross-versions.toml

```diff
@@ -93,10 +97,14 @@
       {{ end }}
 
   [groups.build_config]
-    build_base_image = 'golang:1.17-buster'
+    build_base_image = 'golang:1.18-buster'
     modfile = "go.v0.20.mod"
 
   [groups.build_config.dockerfile_extensions]
     # deal with dependency changes in master until we create the new vx.y.z instance
-    pre_build          = "RUN cd ${PLAN_DIR} && go mod download github.com/libp2p/go-libp2p && go mod tidy"
+    pre_build = """
+RUN cd ${PLAN_DIR} && \
+    go mod download github.com/libp2p/go-libp2p && \
+    go mod tidy -compat=1.18
```

Comment:
Why is this needed? go-libp2p should build with Go 1.17.

Reply:
I thought we wanted to test the current/latest version: https://github.com/libp2p/test-plans/pull/23#discussion_r911688297

Should we test master with 1.18 or 1.17?

Reply:
(sync'd over Slack, using 1.18 was correct)